errbot/core_plugins/plugins.py

```diff
@@ -17,24 +19,27 @@ def repos_install(self, mess, args):
         or an url towards a tar.gz archive : http://www.gootz.net/plugin-latest.tar.gz
         """
         if not args.strip():
-            return "You should have an urls/git repo argument"
-        self._bot.repo_manager.install_repo(args)
-
-        errors = self._bot.plugin_manager.update_dynamic_plugins()
-        if errors:
-            self.send(mess.frm, 'Some plugins are generating errors:\n' + '\n'.join(errors),
-                      message_type=mess.type)
-        else:
-            self.send(
-                mess.frm,
-                ("A new plugin repository has been installed correctly from "
-                 "%s. Refreshing the plugins commands..." % args),
-                message_type=mess.type
-            )
-        loading_errors = self._bot.plugin_manager.activate_non_started_plugins()
-        if loading_errors:
-            return loading_errors
-        return "Plugins reloaded without any error."
+            yield "You should have an urls/git repo argument"
```

Comment:
Since we're changing all this code anyway, lets clarify this error message to *"Please specify a repository listed in '!repos' or give me the URL to a git repository that I should clone for you."*